newsletter/august-2023.md

```diff
@@ -74,7 +74,7 @@ seo:
 *   **[MT Summit 2023](https://machinetranslate.org/mtsummit2023)** - 4 - 8 September - Macau 🇲🇴
 *   **[WAT 2023](https://machinetranslate.org/wat2023)** - 4 September - Macau 🇲🇴
 *   **[CoCo4MT 2023](https://machinetranslate.org/coco4mt-2)** - 4 - 5 September - Macau 🇲🇴
-*   **Seattle area localization meetup** - 13 September - Seattle 🇺🇸
+*   **Seattle area localisation meetup** - 13 September - Seattle 🇺🇸
```

Comment:
I'm not sure about changing the event name to conform with UK spelling.

Reply:
In that case, shouldn't it be written with capital letters **Seattle Area Localization Meetup**? This way, it won't be detected.

Reply:
Yes, we should write `Seattle Localization User Group meetup`